scripts/gateway-restart-when-online.sh

```
@@ -142,6 +146,125 @@ positive_int() {
+# as a new recovery though the traffic path had not moved, and a DHCP NAK that
+# moved only the box's own address read as the same one though every socket had
+# just died.
+#
+# An unanswerable key (no route at all, `ip` missing, a non-IP probe target)
+# is never equal to a recorded one, so it stands down from nothing: this
+# degrades to the unfixed behaviour rather than to a blanket cooldown.
+route_key() {
+  local dest
+  # Same target the probe above uses, so the key describes the path that was
+  # actually proven.
+  set -- ${CLAWBOX_PING_TARGETS:-8.8.8.8}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)